test/common/stats/thread_local_store_test.cc

```diff
@@ -861,8 +861,8 @@ TEST(StatsThreadLocalStoreTestNoFixture, MemoryWithoutTls) {
   TestUtil::forEachSampleStat(
       1000, [&store](absl::string_view name) { store.counter(std::string(name)); });
   const size_t million = 1000 * 1000;
-  EXPECT_MEMORY_EQ(memory_test.consumedBytes(), 19602128); // June 13, 2019
-  EXPECT_MEMORY_LE(memory_test.consumedBytes(), 20 * million);
+  EXPECT_MEMORY_EQ(memory_test.consumedBytes(), 17432336); // June 23, 2019
```

Comment:
Heh... It's tough to keep this date right. And you can't really use a SHA, because you don't know it until the PR is merged...